src/couch_replicator/src/couch_replicator_scheduler.erl

```diff
@@ -1479,6 +1481,31 @@ t_job_summary_crashing_many_times() ->
     end).
 
 
+t_job_summary_proxy_fields() ->
+    ?_test(begin
+        Job =  #job{
+            id = job1,
+            history = [started(10), added()],
+            rep = #rep{
+                source = #httpdb{
+                    url = "https://s",
+                    proxy_url = "http://u:p@sproxy:12"
+                },
+                target = #httpdb{
+                    url = "http://t",
+                    proxy_url = "socks5://u:p@tproxy:34"
+                }
+            }
+        },
+        setup_jobs([Job]),
+        Summary = job_summary(job1, ?DEFAULT_HEALTH_THRESHOLD_SEC),
+        ?assertEqual(<<"http://u:*****@sproxy:12">>,
+            proplists:get_value(source_proxy, Summary)),
+        ?assertEqual(<<"socks5://u:*****@tproxy:34">>,
```

Comment:
this is failing for me:
```
  couch_replicator_scheduler:1485: t_job_summary_proxy_fields...*failed*
in function couch_replicator_scheduler:'-t_job_summary_proxy_fields/0-fun-1-'/1 (src/couch_replicator_scheduler.erl, line 1505)
in call from eunit_test:run_testfun/1 (eunit_test.erl, line 71)
in call from eunit_proc:run_test/1 (eunit_proc.erl, line 510)
in call from eunit_proc:with_timeout/3 (eunit_proc.erl, line 335)
in call from eunit_proc:handle_test/2 (eunit_proc.erl, line 493)
in call from eunit_proc:tests_inorder/3 (eunit_proc.erl, line 435)
in call from eunit_proc:with_timeout/3 (eunit_proc.erl, line 325)
in call from eunit_proc:run_group/2 (eunit_proc.erl, line 549)
**error:{assertEqual,[{module,couch_replicator_scheduler},
              {line,1505},
              {expression,"proplists : get_value ( target_proxy , Summary )"},
              {expected,<<"socks5://u:*****@tproxy:34">>},
              {value,<<"socks5://u:p@tproxy:34">>}]}
  output:<<"">>
```

Reply:
I think it might be an issue about having to compile couch app separately with `make` before running `make eunit` 

The tests did pass locally and I see they are passing in Travis as well:

```
couch_replicator_scheduler:1485: t_job_summary_proxy_fields...[0.009 s] ok
...
```